be/src/formats/parquet/group_reader.h

```diff
@@ -121,11 +118,11 @@ class GroupReader {
     // conjunct ctxs that eval after chunk is dict decoded
     std::vector<ExprContext*> _left_conjunct_ctxs;
 
-    // dict filter column
-    std::vector<GroupReaderParam::Column> _dict_filter_columns;
-    // direct read conlumns
-    std::vector<GroupReaderParam::Column> _direct_read_columns;
-
+    // dict filter column index
+    std::vector<int> _dict_filter_column_indices;
+    std::vector<bool> _use_as_dict_filter_column;
+    // direct read conlumn index
+    std::vector<int> _direct_read_column_indices;
```

Comment:
What is the purpose of this change? To reduce memory and copy overhead? It seems it is not related with the bug?